### ISSUE TITLE
Unable to Complete AR build : 'NoneType' object is not callable

### ISSUE THREAD

**Issue by pavneeta** [Azure/azure-cli#18857]
## Describe the bug

**Command Name**
`az acr build`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'NoneType' object is not callable
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 657, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/acr/build.py", line 109, in acr_build
    variant=platform_variant
TypeError: 'NoneType' object is not callable

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr build --resource-group {} --registry {} --image {} {}`

## Environment Summary
```
Linux-5.4.0-1051-azure-x86_64-with-debian-10.2 (Cloud Shell)
Python 3.6.10
Installer: DEB

azure-cli 2.26.0

Extensions:
ai-examples 0.2.5
ssh 0.1.5

```
## Additional Context

I am trying to follow the MS learn course on AKS - https://docs.microsoft.com/en-us/learn/modules/aks-workshop/03-deploy-container-registry

**Comment by yonzhan** [comment]
route to service team

**Comment by daniel347x** [comment]
I am also having this problem - Ugh - I cannot build my Docker containers...

**Comment by johnynfulleffect** [comment]
Same issue.  cannot use az acr to build

**Comment by luisdlp** [comment]
A hotfix for the cli has been released. Please use Version 2.26.1 for the cli:
https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?tabs=azure-cli#july-14-2021

**Comment by pavneeta** [comment]
I am using the  Azure Cloud Shell - Can I upgrade the Azure CLI in cloud shell to the 2.26.1 version? It seems to be failing. 

`pavneet@Azure:~$ az upgrade
This command is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Your current Azure CLI version is 2.26.0. Latest version available is 2.26.1.
Please check the release notes first: https://docs.microsoft.com/cli/azure/release-notes-azure-cli
Do you want to continue? (Y/n): y
az upgrade is not supported in Cloud Shell.`
